src/bin/nydus-image/main.rs

```diff
@@ -374,42 +374,96 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
+                        )
+                        .arg(arg_chunk_dict.clone())
+                        .arg(arg_prefetch_policy.clone())
+                        .arg(arg_output_json.clone())
+                        .arg(
+                            Arg::new("blob-digests")
+                                .long("blob-digests")
+                                .required(false)
+                                .help("RAFS blob digest list separated by comma"),
+                        )
+                        .arg(
+                            Arg::new("original-blob-ids")
+                                .long("original-blob-ids")
+                                .required(false)
+                                .help("original blob id list separated by comma, it may usually be a sha256 hex string"),
+                        )
+                        .arg(
+                            Arg::new("blob-sizes")
+                                .long("blob-sizes")
+                                .required(false)
+                                .help("RAFS blob size list separated by comma"),
+                        )
+                        .arg(
+                            Arg::new("blob-toc-digests")
+                                .long("blob-toc-digests")
+                                .required(false)
+                                .help("RAFS blob toc digest list separated by comma"),
+                        )
+                        .arg(
+                            Arg::new("blob-toc-sizes")
+                                .long("blob-toc-sizes")
+                                .required(false)
+                                .help("RAFS blob toc size list separated by comma"),
+                        )
+                        .arg(arg_config.clone())
+                        .arg(
+                            Arg::new("SOURCE")
+                                .help("bootstrap paths (allow one or more)")
+                                .required(true)
+                                .num_args(1..),
+                        )
+                        .arg(
+                            Arg::new("digester")
+                                .long("digester")
+                                .help("Algorithm to digest data chunks:")
+                                .required(false)
+                                .default_value("blake3")
+                                .value_parser(["blake3", "sha256"]),
+                        )
+                        .arg(
+                            Arg::new("verbose")
+                                .long("verbose")
+                                .short('v')
+                                .help("Output message in verbose mode")
+                                .action(ArgAction::SetTrue)
+                                .required(false),
+                        )
+                        .arg(
+                            Arg::new("features")
```

Comment:
Which of these parameters above will actually be used?

Reply:
Done